lib/WebServer/ConfigWebServer.cpp

```diff
@@ -40,7 +41,13 @@ void ConfigWebServer::begin(Settings& settings) {
   updateSettings(settings);
 
   server.on("/", authenticated([this]() {
-              server.send_P(200, TEXT_PLAIN, PSTR("Hello from rfESP"));
+              server.sendHeader(F("Content-Encoding"), "gzip");
+              server.sendHeader(F("Content-Length"), String(index_html_gz_len));
```

Comment:
Why `sendHeader(F("Content-Length")` and then `setContentLength(CONTENT_LENGTH_UNKNOWN)` looks like an ugly workaround.  If it is a workaround, please describe the steps with comments.

Reply:
@janLo any comment here?

Reply:
I had this from https://github.com/sidoh/esp8266_milight_hub/blob/master/lib/WebServer/MiLightHttpServer.cpp#L469-L475.

Good if it isnnt necessary.